microros_esp32_extensions/sdkconfig.defaults

```diff
@@ -0,0 +1,3 @@
+CONFIG_ESP_MAIN_TASK_STACK_SIZE=25000
+CONFIG_FREERTOS_UNICORE=y
+CONFIG_ESP_TASK_WDT=n
```

Comment:
Hello @donRaphaco, do you have any idea on why is this necessary?

Reply:
The `CONFIG_ESP_TASK_WDT=n` setting I added to prevent getting a warning (during runtime) that the watchdog timer for the main task did not reset. I figured task watchdog timers aren't really useful if there is only one task, hence I deactivated them with this setting. ([about the esp32 task watchdog timer](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/wdts.html#task-watchdog-timer))

The setting `CONFIG_FREERTOS_UNICORE=y` I don't know why it's necessary for microros, but if I don't use this setting I'm getting a runtime error and I can't really figure out why exactly.

Reply:
Thank you very much @donRaphaco

Reply:
@pablogs9 I just realized, that the error I had with `CONFIG_FREERTOS_UNICORE=y` is now solved somehow, which is really cool since I noticed a performance boost right away when I removed the setting.
So `CONFIG_FREERTOS_UNICORE=y` isn't actually necessary any more and should be removed :)

Reply:
Thank you very much @donRaphaco. Could you share a reference link and I will commit the changes?

BTW, have you seen this: https://github.com/micro-ROS/micro_ros_espidf_component ?

Reply:
@pablogs9 what do you mean by "reference link"?

Yeah I've seen https://github.com/micro-ROS/micro_ros_espidf_component and tested it. I really like it, since it's better integrates with the esp-idf stuff 👍

Reply:
sorry, I meant a link to somewhere where is explained which was this issue with `CONFIG_FREERTOS_UNICORE` and why /how they have solved it.

Reply:
Well to be honest I never really understood the issue with `CONFIG_FREERTOS_UNICORE`.
All I know is that in the first PR #12 I made, without this setting there was a runtime error even before the `app_main` function was executed.
I have no idea what the actual problem was then and also don't know when and how it was fixed.
So sorry I don't have a reference for this.

Reply:
Ok, so I will try both ESP32 + micro-ROS build tools and I will remove it from config defaults